pom.xml

```diff
@@ -542,6 +542,21 @@
             <artifactId>dataverse-spi</artifactId>
             <version>2.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>javax.cache</groupId>
+            <artifactId>cache-api</artifactId>
+            <version>1.1.1</version>
+        </dependency>
```

Comment:
Unused dependency, can be removed. (If not: remove version and add scope provided)

```suggestion
```